Website/src/rendering/src/pages/_app.tsx

```diff
@@ -61,7 +69,7 @@ function App({ Component, pageProps, router }: AppProps): JSX.Element {
 
       {/* DEMO TEAM CUSTOMIZATION - CDP/Discover integration. It is important this script is rendered before the <Component> so the CDP calls made on the first page load are successful. */}
       {CdpScripts}
-      {MerchandisingScripts}
+      {isCommerceEnabled ? MerchandisingScripts : null}
```

Comment:
The condition should be moved to the `src/services/MerchandisingService` file to simplify all the places we could use the MerchandisingScripts.